Makefile

```diff
@@ -1,9 +1,13 @@
-HEADERS := -I"src/main/headers" -I"${JAVA_HOME}\include" -I"${JAVA_HOME}\include\win32"
+#HEADERS := -I"src/main/headers" -I"${JAVA_HOME}\include" -I"${JAVA_HOME}\include\win32" -I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\cppwinrt" -I"C:\Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.41.34120/include"
+HEADERS := /I"src/main/headers" /I"${JAVA_HOME}\include" /I"${JAVA_HOME}\include\win32" /I"C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\cppwinrt" /I"C:\Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.41.34120/include"
 SOURCES := $(wildcard src/main/native/*.cpp)
 
 ########
 
 all: install
 
+#install:
+#	g++ -Wall -D_JNI_IMPLEMENTATION_ -Wl,--kill-at $(HEADERS) -shared -osrc/main/resources/integrations.dll $(SOURCES) -lcrypt32 -lshell32 -lole32 -luuid -luser32 -lwindowsapp
+
 install:
-	g++ -Wall -D_JNI_IMPLEMENTATION_ -Wl,--kill-at $(HEADERS) -shared -osrc/main/resources/integrations.dll $(SOURCES) -lcrypt32 -lshell32 -lole32 -luuid
\ No newline at end of file
+	cl /EHsc /std:c++17 /LD /Fe:src/main/resources/integrations.dll $(HEADERS) $(SOURCES) /link crypt32.lib shell32.lib ole32.lib uuid.lib user32.lib windowsapp.lib
```

Comment:
I took over these suggestions, but:

`/sdl` can't be added, otherwise `WinAppearance_Native.cpp` does not compile any more:
```
src/main/native/org_cryptomator_windows_uiappearnce_WinAppearance_Native.cpp(23): error C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
src/main/native/org_cryptomator_windows_uiappearnce_WinAppearance_Native.cpp(34): error C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
src/main/native/org_cryptomator_windows_uiappearnce_WinAppearance_Native.cpp(40): error C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
```

`/WX` can't be added, otherwise the library does not compile at all.